package.json

```diff
@@ -27,10 +27,13 @@
     "restify": "2.8.2"
   },
   "devDependencies": {
+    "eslint-config-fxa": "^1.4.0",
     "grunt": "0.4.5",
     "grunt-cli": "0.1.13",
     "grunt-contrib-copy": "0.5.0",
+    "grunt-eslint": "^15.0.0",
     "jsxgettext-recursive": "0.0.5",
+    "load-grunt-tasks": "^3.2.0",
```

Comment:
Good catch.

**PROTIP:** You can remove the semver range operators with the `--save-exact` flag when doing `npm install`, or I hacked my ~/.npmrc with `save-exact=true` so it always pins to the specific version.

Or in this case just manually remove those `^`s.